.eslintignore

```diff
@@ -1 +1,2 @@
 src/resources/js/**/*.min.js
+src/modules/**/__tests__/*
```

Comment:
```suggestion
src/modules/**/__tests__/*
tests/**/*.js
```

Reply:
Honestly, the first (existing) line could likely be `/**/*.min.js` because we don't ever want to lint minimized files, wherever they are.